examples/103_transformer_layout_options.ipynb

```diff
@@ -0,0 +1,252 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 🦌 ELK Transformer 🤖\n",
+    "\n",
+    "A transformer object that will convert some input source into valid Elk Json. This\n",
+    "example using a transformer around a networkx graph."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import pathlib\n",
+    "\n",
+    "import ipywidgets\n",
+    "import networkx\n",
+    "import traitlets\n",
+    "from IPython.display import JSON\n",
+    "\n",
+    "import ipyelk\n",
+    "import ipyelk.nx\n",
+    "import ipyelk.tools\n",
+    "from ipyelk.diagram import layout_options"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "l = ipyelk.nx.XELKTypedLayout()"
```

Comment:
lowercase L is a singularly bad variable name

Reply:
fixed this and added some other minor wording cleanup to that example notebook. I created a followon issue for more examples and the importnb PR should be there.